main.tf

```diff
@@ -218,7 +218,7 @@ resource "google_compute_firewall" "istioctl_firewall" {
   allow {
     # see https://istio.io/latest/docs/setup/platform-setup/gke/
     protocol = "tcp"
-    ports    = ["10250", "443", "15017"]
+    ports    = ["10250", "443", "15017", "8080", "15000"]
```

Comment:
May i suggest the following

```terraform
ports = [
          "10250", "443", "15017", # see https://istio.io/latest/docs/setup/platform-setup/gke/
          "8080", "15000", # see https://kiali.io/documentation/latest/installation-guide/#_google_cloud_private_cluster_requirements
        ]
```

Reply:
@syedrakib great, can you add it just like that, or do I have to change the MR?

Reply:
As the author of the PR, i guess you have to update it